index.d.ts

```diff
@@ -21,23 +21,23 @@ declare namespace ora {
 
 	interface Options {
 		/**
-		Text to display after the spinner.
-		*/
+		 Text to display after the spinner.
+		 */
```

Comment:
Don't do unrelated changes.

Reply:
Sorry, I think that happened when I changed from spaces to tab in my vscode.